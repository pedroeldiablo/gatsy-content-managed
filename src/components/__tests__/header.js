import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../header'

describe('Header', () => {
  const id = 'test'
  const name = 'myInput'
  const value = 'selected'
  const label = 'hello'
  const data = jest.fn()

  const props = {
    id,
    name,
    value,
    label,
    data,
    checked: false,
  }

  it('renders correctly', () => {
    const tree = renderer.create(<Header {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
