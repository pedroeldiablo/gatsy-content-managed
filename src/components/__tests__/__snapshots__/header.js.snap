// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="sc-AxjAm gobpuM"
>
  <nav
    className="sc-AxirZ bvspZ"
  >
    <ul
      className="navbar"
    >
      <li>
        <a
          className="link"
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/about"
        >
          About
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/blog"
        >
          Blog
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/article"
        >
          Articles
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
